doc/release/1.14.1-notes.rst

```diff
@@ -2,38 +2,91 @@
 NumPy 1.14.1 Release Notes
 ==========================
 
-This is a bugfix release for some problems found since 1.14.0. The most
-important fixes are the reversion of the multifield index "view" behavior (see
-compatibility notes), and fixes to the spacing in the str and repr of complex
-values.
+This is a bugfix release for some problems reported following the 1.14.0 release. The major
+problems fixed are the following.
 
-The Python versions supported are 2.7 and 3.4 - 3.6. The Python 3.6 wheels
-available from PIP are built with Python 3.6.2 and should be compatible with
-all previous versions of Python 3.6. It was cythonized with Cython 0.26.1,
-which should be free of the bugs found in 0.27 while also being compatible with
-Python 3.7-dev.
-
-
-Compatibility notes
-===================
-
-Multifield Indexing of Structured Arrays will still return a copy
------------------------------------------------------------------
-We have reverted the change in 1.14.0 that multi-field indexing of structured
-arrays returns a view instead of a copy, and pushed it back to 1.15. This will
-give us time to implement additional related bugfixes to ease the transition.
-Affected users should read the the 1.14.1 Numpy User Guide for advice on how to
-manage this transition, under "basics/structured arrays/accessing multiple
-fields". A new method `numpy.lib.recfunctions.repack_fields` has been
-introduced to help with this.  All other changes to structured arrays described
-in the 1.14.0 release notes still hold in 1.14.1, and have not been reverted.
+* Problems with the new array printing, particularly the printing of complex
+  values, Please report any additional problems that may turn up.
+* Problems with ``np.einsum`` due to the new ``optimized=True`` default. Some
+  fixes for optimization have been applied and ``optimize=False`` is now the
+  default.
+* The sort order in ``np.unique`` when ``axis=<some-number>`` will now always
+  be lexicographic in the subarray elements. In previous NumPy versions there
+  was an optimization that could result in sorting the subarrays as unsigned
+  byte strings.
+* The change in 1.14.0 that multi-field indexing of structured arrays returns a
+  view instead of a copy has been reverted but remains on track for NumPy 1.15.
+  Affected users should read the 1.14.1 Numpy User Guide section
+  "basics/structured arrays/accessing multiple fields" for advice on how to
```

Comment:
Can we get an rst link for this, so that the sphinx release note page links to it?

Reply:
Unfortunately those docs are inside a `.py` file: `numpy/doc/structured_arrays.py`, so I'm not sure that will work out well.

https://github.com/numpy/numpy/blob/maintenance/1.14.x/numpy/doc/structured_arrays.py

Reply:
Let's not over design this, it's just a release note. The documentation will go up after the release.

Reply:
I'd like to get rid of the `*.py` documentation files at some point. It is confusing to have the documentation scattered about like that.